src/validation_workflow/api_test_cases.py

```diff
@@ -30,6 +30,11 @@ def test_apis(projects: list) -> Any:
             ['https://localhost:9200/_cat/plugins?v', 200, ''],
             ['https://localhost:9200/_cat/health?v', 200, 'green'],
         ]
+        # Use http in the request url if the security plugin is absent
+        if not security_plugin_exists:
+            for api in test_apis:
+                api[0] = "http" + api[0][5:]  # type: ignore
```

Comment:
I understand what this is now, as @Divyaasm is trying to replace the first 5 char of `https` with `http`.

In that case, it is better to do a string concatenation such as:

```
protocol_prefix = "https" if security_plugin_exisits else "http"
test_apis = protocol_prefix + body

```